overlay.d/05rhcos/usr/lib/systemd/system/gcp-routes.service

```diff
@@ -6,8 +6,8 @@ After=network.target
 
 [Service]
 Type=simple
-ExecStart=/bin/bash /usr/sbin/gcp-routes.sh start
-ExecStopPost=/bin/bash /usr/sbin/gcp-routes.sh cleanup
+ExecStart=/usr/sbin/gcp-routes.sh start
+ExecStopPost=/usr/sbin/gcp-routes.sh cleanup
```

Comment:
Kinda odd to have something in `/usr/sbin` use `.sh`. If this isn't something intended to be used directly by users, it should be in `/usr/libexec` instead. Anyway, not new so definitely not a blocker!